test/commands/notification/create_test.rb

```diff
@@ -16,4 +18,15 @@ class Notifications::CreateTest < ActiveSupport::TestCase
     assert_equal "#{user.id}-mentor_started_discussion-Discussion##{discussion.id}", notification.anti_duplicate_key
     assert_equal params, notification.send(:params)
   end
+
+  test "broadcasts message" do
+    user = create :user
+    type = :mentor_started_discussion
+    discussion = create(:solution_mentor_discussion)
```

Comment:
Let's not use brackets for factory creation. Maybe there's a rubocop rule we could add or something? And something to add to the style guide.